geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/RedisStatsIntegrationTest.java

```diff
@@ -474,27 +474,27 @@ public void NetworkKiloBytesReadOverLastSecond_shouldReturnCorrectData() {
   @Test
   public void clientsStat_withConnectAndClose_isCorrect() {
 
-    jedis = new Jedis("localhost", server.getPort(), TIMEOUT);
-    jedis.ping();
+    Jedis jedis2 = new Jedis("localhost", server.getPort(), TIMEOUT);
```

Comment:
What is this change for? Seems unrelated to the PR.

Reply:
> While testing, I noticed that there was a problem with the stat for `connected clients`. Occasionally this test and the next one would fail. My fix seems to have addressed the issue.

Reply:
I doubt you ran it enough times since it's clearly not the correct fix. The real problem is that when the connection is `closed` the stat which counts the connections is updated asynchronously. This means that if you read the stat immediately after a `close` it may not be correct. You need to wait for the stat to 'settle' before using it.

Reply:
> I'm not sure I follow you. I've run this fix a bunch of times on my laptop and it always passes.

Reply:
Take a look at the `before` method. When `close` is called in there, you cannot assume that the stat will immediately be updated. You need to `wait` at that point. I'm not sure how much more detail you need to understand.

Reply:
Maybe point me at a failing test and prove your point for once, instead of nitpicking and making condescending comments. No wonder everything around here takes 3x as long as it should if we have people like you gumming up the works.